app/views/edit/EditCheckYourAnswersView.scala.html

```diff
@@ -53,7 +53,12 @@
 
         <div class="govuk-button-group">
             @button("edit-cya-button")
-            @p(link(location = controllers.routes.ViewAuthorityController.onPageLoad(accountId, authorityId).url, linkMessage = "edit-cya-cancel"))
+            @p(link(
```

Comment:
no need to wrap with `@p` since the link component has a `P` tag. Below suggestion should be fine 
```suggestion
          @link(
              location = controllers.routes.ViewAuthorityController.onPageLoad(accountId, authorityId).url, 
              linkMessage = "edit-cya-cancel",
              ariaLabel = Some(messages("edit-cya-cancel"))
           )
```